### PR TITLE
Seabeta 244 govuk update

### PR DESCRIPTION
**Pull request by CodrinCotarlanSoftwire**

### CODE REVIEW

File: SeaPublicWebsite/Views/EnergyEfficiency/ConfirmAddress.cshtml [hunk 1]
```diff
@@ -55,10 +55,16 @@
...
             else
             {
-                string error = Model.GetErrorFor(m => m.SelectedEpcId);
+                string error = null;
+                var errors = ViewData.ModelState[nameof(ConfirmAddressViewModel.SelectedEpcId)].Errors;
+                if (errors.Count > 0)
+                {
+                    error = errors[0].ErrorMessage;
+                }
+                // string error = Model.GetErrorFor(m => m.SelectedEpcId);
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/ConfirmAddress.cshtml:64]: Please delete this commented out code

File: SeaPublicWebsite/Views/EnergyEfficiency/BungalowType.cshtml [hunk 2]
```diff
@@ -24,7 +25,7 @@
         
         <form action="@Url.Action("BungalowType_Post", "EnergyEfficiency", new {reference = Model.Reference, change = Model.Change})" method="post" novalidate>
             
-            @(Html.GovUkRadiosFor(m => m.BungalowType,
+            @(await Html.GovUkRadiosFor<BungalowTypeViewModel, BungalowType>(m => m.BungalowType,
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/BungalowType.cshtml:28]: Now that this is building correctly can we delete the explicit typing (`<BungalowTypeViewModel, BungalowType>`) here and in all the other `GovUkRadiosFor` calls?

**Review by DanCorderSoftwire — CHANGES_REQUESTED** [review]

File: SeaPublicWebsite/Views/EnergyEfficiency/ConfirmAddress.cshtml (hunk 1 above)
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite/Views/EnergyEfficiency/ConfirmAddress.cshtml]: Done

File: SeaPublicWebsite/Views/EnergyEfficiency/BungalowType.cshtml (hunk 2 above)
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite/Views/EnergyEfficiency/BungalowType.cshtml]: Yup! Those are not needed anymore, I removed them.

**Review by DanCorderSoftwire — APPROVED** [review]

**Pull request merged**